demos/classification_benchmark_demo/cpp/main.cpp

```diff
@@ -149,7 +141,6 @@ int main(int argc, char *argv[]) {
                 i--;
             } else {
                 readerMetrics.update(readingStart);
-                // Clone cropped image to keep memory layout dense to enable -auto_resize
```

Comment:
Why is it removed?

Reply:
I removed all mentions of auto resize, but I think this can be restored.

Reply:
Yes, we steel need dense mats

Reply:
Restored.